tests/puppeteer/pages/Install/install.js

```diff
@@ -125,7 +125,7 @@ module.exports = class Install extends CommonPage {
     await this.page.click(this.dbLoginInput, {clickCount: 3});
     await this.page.type(this.dbLoginInput, global.db_user);
     await this.page.click(this.dbPasswordInput, {clickCount: 3});
-    await this.page.type(this.dbPasswordInput, global.db_passwd);
+    await this.page.type(this.dbPasswordInput, global.db_passwd.replace('empty_string', ''));
```

Comment:
What for ?

Reply:
travis has empty value for db_passwd, if we passe `DB_PASSWD=""`, he will take admin from global
```js
   global.db_passwd = process.env.DB_PASSWD || 'admin';
```
So I pass a value, and replace it with an empty string in test

Reply:
Nice workaround.

Reply:
So this code only works on Travis? :thinking: 
We're able to set up DB_PASSWD global variable especially for Travis.
I'm not in favor of this durty fix :/

Reply:
Maybe you can do
```
global.db_passwd = process.env.DB_PASSWD || '';
```

as default, because many instances (or many devs) have no password for database connection.
Because i'm fuc**d if I have `empty_string` in my password :trollface:

Reply:
Admin was set as db_password for docker 
so I change it @PierreRambaud 
https://github.com/PrestaShop/PrestaShop/pull/15556/commits/f43d49ae73c2971d0ce8da237950330c2c7b9fef